src/rtcm3.c

```diff
@@ -166,7 +166,7 @@ static void adjweek(rtcm_t *rtcm, double tow)
 {
     double tow_p;
     int week;
-    
+
```

Comment:
Lot of white space changes might be a distraction. We might be moving to automated formatting to ease these issues. Would you be able to just remove the bulk of these for now.